src/apiclass.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable import/no-cycle */
+
+import { displayEpisodes, displayComments } from './displayItems.js';
+
+const ourID = 'VquE0tA2WmPqHydxagQA';
+
+const episodes = ' https://api.tvmaze.com/seasons/1/episodes';
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps';
```

Comment:
- The web app uses an API that bases its development around it ✔
- The API should allow for getting a list of items with a unique item id and detailed information about the items ✔
- The project uses the Involvement API to record the different user interactions (likes and comments) 📝
 
Well done 🙌🔥